### DISCUSSION TITLE
Streamlit version support + upgrade

### DISCUSSION THREAD

**Discussion by ACR-GH** [whitphx/stlite#650] [Q&A]
Hi there. Is there anyway the underlying Streamlit version that comes packaged with stlite can be upgraded? Current version is 1.24.0 while current Streamlit version is 1.28.x with important updates such as st.connection(), placeholder parameter and improvements to data_editor and expander components.

Is there an upgrade path planned for stlite?

**Comment by whitphx** [comment]
Unfortunately the underlying Streamlit version is tightly coupled to stlite's implementation.
I have a plan to upgrade it.
Thank you for your patience.

**Comment by whitphx** [comment]
Now that's upgraded.
